src/UI/LocationListView.re

```diff
@@ -83,17 +75,9 @@ let item =
     );
 
   let locationWidth =
-    switch (Revery.UI.getActiveWindow()) {
-    | Some(window) =>
-      Revery.Draw.Text.measure(
-        ~window,
-        ~fontSize=uiFont.fontSize,
-        ~fontFamily=uiFont.fontFile,
-        locationText,
-      ).
-        width
-    | None => String.length(locationText) * uiFont.fontSize
-    };
+    Oni_Core.EditorFont.(
+      int_of_float(editorFont.measuredWidth) * String.length(locationText)
+    );
```

Comment:
This is wrong. `locationText` uses the UI font, not the editorFont.see https://github.com/onivim/oni2/blob/33f1ab892a0720f84830af7178a6c33b0f146db1/src/UI/LocationListView.re#L82-L86

Reply:
Derp, I don't know what i was thinking... tricked by the font I had on my Mac, I guess 😬 

You're right, thanks for catching that!